.gitattributes

```diff
@@ -12,6 +12,13 @@
 /phpunit.xml.dist export-ignore
 /src/Fixer/Internal/ export-ignore
 /tests/ export-ignore
+/feature-or-bug.rst export-ignore
+/logo.md export-ignore
+/logo.png export-ignore
+/CHANGELOG.md export-ignore
```

Comment:
```suggestion
```

i find it useful to have changelog in vendor dir, eg `cat vendor/symfony/console/CHANGELOG.md`